Readme.md

```diff
@@ -3,13 +3,16 @@
 [![npm version](https://badge.fury.io/js/eswrap.svg)](https://badge.fury.io/js/eswrap)
 [![Build Status](https://travis-ci.org/kevinastone/eswrap.svg?branch=master)](https://travis-ci.org/kevinastone/eswrap)
 
-eswrap is a ecmascript formatting utlity.  It will parse a supplied block of code and apply formatting to attempt to fit it within a col width.
+**eswrap** is an ECMAScript formatting utility.
+It attempts to reformat a code block to fit within a specified column count:
 
-```
+```bash
 echo "[variable1, variable2, variable3];" | eswrap --limit 20
-[
+
+# Output:
+'[
   variable1,
   variable2,
   variable3,
-];
+];'
```

Comment:
Oh! That's the closing quote for the string that begins under the `# Output` line... I enclosed it in a string to make the highlighted syntax look uniform (as opposed to shell-script syntax). You can see how [it looks here](https://github.com/Cutlery-Drawer/eswrap/blob/f2cf0128bb5710158ffb1c0f2210e7c2eb05e04e/Readme.md). =)